cpp/tests/community/balanced_edge_test.cpp

```diff
@@ -15,7 +15,7 @@
 #include <rmm/device_vector.hpp>
 #include <rmm/exec_policy.hpp>
 
-TEST(balanced_edge, success)
+TEST(balanced_edge, DISABLED_success)
```

Comment:
@ChuckHastings is currently diagnosing the failure here, which is unrelated to the removal of cugraph-equivariance. We think it is related to https://github.com/rapidsai/raft/pull/2481. I'll open a tracking issue for now.

Reply:
Tracking issue: https://github.com/rapidsai/cugraph/issues/4777